### ISSUE TITLE
[BUG] CSS properties in `.style` are caught by NamePatternRule

### ISSUE THREAD

**Issue by ia-eknorr** [bw-design-group/ignition-lint#11]
**Describe the bug**
CSS properties that use `kebab-case` are flagged by NamePatternRule

```
• root.root.props.style.touch-action: Name 'touch-action' doesn't follow camelCase for property (suggestion: 'touchAction') 
```

**To Reproduce**

1. Download the sample [view.json](https://github.com/user-attachments/files/22703605/view.json)
2. Lint the view with the default `rule_config.json` at the root of ignition-lint`
  `poetry run ignition-lint --files <file-path-to>/view.json`
4. Observe the following warnings
```
⚠️  Found 2 warnings in .eric/services/ignition-frontend/projects/data-center-demo/com.inductiveautomation.perspective/views/Exchange/PanZoomIframe/PanZoomIframe/view.json:
  📋 NamePatternRule (warning):
    • root.root.children[0].Iframe.props.style.touch-action: Name 'touch-action' doesn't follow camelCase for property (suggestion: 'touchAction')
    • root.root.props.style.touch-action: Name 'touch-action' doesn't follow camelCase for property (suggestion: 'touchAction')
```

**Expected behavior**
All valid CSS properties should be allowed by the NamePatternRule, regardless of their naming convention.
